src/game.cpp

```diff
@@ -9239,15 +9251,18 @@ int game::list_items(const int iLastState)
     //Area to search +- of players position.
     const int iRadius = 12 + (u.per_cur * 2);
 
+    bool bRadiusSort = true;
+
     //this stores the items found, along with the coordinates
-    std::vector<map_item_stack> ground_items = find_nearby_items(iRadius);
+    std::vector<map_item_stack> ground_items_radius = find_nearby_items(iRadius);
+    std::vector<map_item_stack> ground_items = ground_items_radius;
     //this stores only those items that match our filter
     std::vector<map_item_stack> filtered_items = (sFilter != "" ?
             filter_item_stacks(ground_items, sFilter) :
             ground_items);
     int highPEnd = list_filter_high_priority(filtered_items, list_item_upvote);
     int lowPStart = list_filter_low_priority(filtered_items, highPEnd, list_item_downvote);
-    const int iItemNum = ground_items.size();
+    int iItemNum = ground_items.size();
```

Comment:
With the categories sort active the size of iItemNum gets bigger by the amount of categories. In distance sort it gets lower again.